utils/rpc/options.go

```diff
@@ -6,19 +6,23 @@ package rpc
 import (
 	"net/http"
 	"net/url"
+
+	"golang.org/x/exp/maps"
 )
 
 type Option func(*Options)
 
 type Options struct {
 	headers     http.Header
 	queryParams url.Values
+	cookies     map[string]*http.Cookie
```

Comment:
Can this just be a `[]*http.Cookie`? Seems like we never use the keys in this map